components/lands/scene.tsx

```diff
@@ -59,6 +60,7 @@ export const Scene: FunctionComponent<SceneProps> = ({
 
   const bind = useGesture({
     onDrag: ({ first, down, pinching, cancel }) => {
+      if (window.innerWidth < 768) return;
```

Comment:
in css files we consider mobile as 740px and here 768px. They should have the same value for mobile no?